locust/USAGE.md

```diff
@@ -0,0 +1,55 @@
+## Purpose 
+This directory contains a Dockerfile that builds the [OpenSSL v3](https://github.com/openssl/openssl)  [OQS provider](https://github.com/open-quantum-safe/oqs-provider), and Python3 which allows locust to negotiate quantum-safe keys in TLS 1.3.
+
+## Quick start
+Assuming Docker is [installed](https://docs.docker.com/install) the following command
+
+2) Run `docker build -t oqs-locust:0.0.1 .` to create a post quantum-enabled Locust docker image.
+3) To verify all components perform quantum-safe operations, first start the container with docker compose `LOGGER_LEVEL=DEBUG HOST=https://qsc-nginx.discovery.hi.inet:4433 docker compose  up --scale master=1 --scale worker=8`.
```

Comment:
Where does this host URL come from?

Reply:
It is my internal OQS host URL, Fix in this [commit](https://github.com/open-quantum-safe/oqs-demos/pull/304/commits/91769ca63c25a4b9fb393634cf6ae2397f87d132)